VideoWeb/VideoWeb.UnitTests/Controllers/ConferenceManagement/DismissWitnessTests.cs

```diff
@@ -0,0 +1,91 @@
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using VideoWeb.Common.Models;
+using VideoWeb.Services.Video;
+using VideoWeb.UnitTests.Builders;
+using ProblemDetails = Microsoft.AspNetCore.Mvc.ProblemDetails;
+
+namespace VideoWeb.UnitTests.Controllers.ConferenceManagement
+{
+    public class DismissWitnessTests : ConferenceManagementControllerTestBase
+    {
+        [SetUp]
+        public void Setup()
+        {
+            TestConference = BuildConferenceForTest();
+        }
+        
+        [Test]
+        public async Task should_return_unauthorised_if_participant_is_not_a_witness()
+        {
+            var participant = TestConference.Participants.First(x => x.Role == Role.Individual);
+            var user = new ClaimsPrincipalBuilder()
+                .WithUsername(participant.Username)
+                .WithRole(AppRoles.CitizenRole).Build();
+
+            Controller = SetupControllerWithClaims(user);
+
+            var result = await Controller.DismissWitnessAsync(TestConference.Id, participant.Id);
+            var typedResult = (UnauthorizedObjectResult) result;
```

Comment:
could assert type before casting .Should().BeOfType<T>()

Reply:
if changing, same in `should_return_accepted_when_participant_is_witness_and_judge_is_in_conference`

Reply:
done